README.md

```diff
@@ -100,11 +100,79 @@ this can be used with `watch`.
 watch -c juju status --color
 ```
 
-### Alternate deployment methods
+## Alternate deployment methods
 
+### Deploying storage
```

Comment:
I don't know that this is necessarily an alternate deployment method... but I also don't know where else I would put this...